gpu/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
```

Comment:
Same, should this file be here?

Reply:
Not sure how the gpu/ directory got there (and I didn't notice it somehow). Thanks, removed.